conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,98 @@
+# --- !Ups
+
+CREATE TABLE paypal(
+id serial PRIMARY KEY,
+email text
+);
+
+CREATE TABLE credit_card(
+id serial PRIMARY KEY,
+card_number int,
+expiration_date DATE
+);
+
+
+CREATE TABLE payments(
+id serial PRIMARY KEY,
+paypal_id int references paypal(id),
```

Comment:
Recimo da u nekoj budućoj evoluciji, ja hoću da izbrišem ovaj `FOREIGN KEY`, kako bih to mogao uraditi?
Svi indexi trebaju da imaju explicitno definisana imena:

```sql
...
paypal_id int,
CONSTRAINT payments_paypal_id FOREIGN KEY (paypal_id) REFERENCES paypal(id)
```

isto tako, običaj je da se defnišu i update i delete akcije

```sql
ON UPDATE CASCADE ON DELETE RESTRICT
```
u slučaju da se vijrednos u paypal abeli promijeni.